gitbook-docs/market/economics.md

```diff
@@ -1,124 +1,183 @@
+The situation when the proof is not provided on time or is not provided at all is considered undesirable.
+Such situations impact the producer's rating in the first place.
+In addition to downgrading the rating, regular non-fulfillment of undertaken obligations may have
+the following consequences:
 
-* Account blocking
-* Limits on the allowed number of assigned bids
-* Setting a limit on the minimum generation time for the producer
+* account blocking;
+* limits on the allowed number of assigned requests;
+* a limit on the minimum generation time for the producer.
 
-If the proof was not provided on time, the bid can be unmatched and reassigned to another proof producer.
+Requests may be unmatched and reassigned to another proof producer if the proof was not submitted on time.
 
-Proof producers do not get the payment for the proofs that were not provided in the requested time interval. However, there is a difference in rating change for the cases when proof was not provided on time and the proof was not provided at all. Remind that proof producer define their minimum proof generation time, and the matching algorithm never matches asks to the bids with tb\<ta.
+Proof producers don't receive payments for proofs that were not provided in the requested time interval.
+However, there's a difference in rating change for when proof was not provided on time
+and when it was not provided at all.
+Remember that proof producers define their minimum proof generation time, and the matching algorithm
+never matches proposals to the requests with _t\_r_ < _t\_p_.
 
-### The delay between the bid/ask submission and the matching
+## The delay between the request/proposal submission and the matching
 
-There is no delay between the bid/ask submitting and running the matching algorithm. It means that as soon as the bid/ask was submitted, Proof Market tries to match it with other existing orders.
+There is no delay between the request/proposal submitting and running the matching algorithm.
+As soon as the request/proposal is submitted, the Proof Market tries to match it with
+other existing orders.
 
-### Penalties
+## Penalties
 
-There are two types of penalties to proof producers:
+There are two types of penalties for proof producers:
 
-* Rating decreasing. It happens when proof was not provided on time or was not provided at all.
-* Fine on the locked funds. It happens when verification fails for the provided proof; then the proof producer pays _pf \* c_ tokens as a fine, where _c_ is the order's cost. It is considered malicious behaviour from the proof producer side.
+* Decreasing the rating.
+  It applies when proof was not provided on time or was not provided at all.
+* A fine on the locked funds.
+  When verification fails for the provided proof, the proof producer pays _pf \* c_ tokens as a fine,
+  where _c_ is the order's cost.
+  This is considered malicious behavior from the proof producer side.
```

Comment:
Unclear what's *this* in the last sentence. The fine?

Overall, this whole block is reversed. We group it by penalties and then explain what can cause such penalties.

What if we talk about misbehavior first?

* If a proof producer provides the proof out of time or doesn't provide it at all, their rating will decrease.
* If a proof producer provides proof that doesn't verify, a fine of _pf \* c_ tokens will be subtracted from their deposit, where _c_ is the order's cost.